qiskit/pulse/transforms/alignments.py

```diff
@@ -12,24 +12,22 @@
 """A collection of passes to reallocate the timeslots of instructions according to context."""
 
 import abc
-from typing import Callable, Dict, Any, Union
+from typing import Callable, Dict, Any, Union, Tuple
 
 import numpy as np
 
-from qiskit.circuit.parameterexpression import ParameterExpression
+from qiskit.circuit.parameterexpression import ParameterExpression, ParameterValueType
 from qiskit.pulse.exceptions import PulseError
 from qiskit.pulse.schedule import Schedule, ScheduleComponent
-from qiskit.pulse.utils import instruction_duration_validation
+from qiskit.pulse.utils import instruction_duration_validation, deprecated_functionality
 
 
 class AlignmentKind(abc.ABC):
     """An abstract class for schedule alignment."""
 
-    is_sequential = None
```

Comment:
for review; attribute is removed because user can override this, then we should have this in QPY but it's unnecessary complexity. This is designed to be static value, so it's now replaced by subclass which user cannot override.